web/src/components/shared/SubNavBar/SubNavBar.jsx

```diff
@@ -22,12 +22,11 @@ export default function SubNavBar({
   }
 
   let configSequence = app?.downstream?.currentVersion?.parentSequence;
-  let kotsSequence = app?.downstream?.currentVersion?.parentSequence;
+  let kotsSequence = app?.currentSequence;
```

Comment:
this now means that the config tab goes to latest version not current, no?

Reply:
config sequence is on line 24